seldon/seldon-core-operator/base/resources.yaml

```diff
@@ -0,0 +1,3656 @@
+---
+# Source: seldon-core-operator/templates/webhook.yaml
+apiVersion: v1
+data:
+  ca.crt: 'LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lSQVBXU29tTUhRL1BmcUpMODNxa3RRZ0F3RFFZSktvWklodmNOQVFFTEJRQXcKSERFYU1CZ0dBMVVFQXhNUlkzVnpkRzl0TFcxbGRISnBZM010WTJFd0hoY05NakF3TVRJeE1UQTFOVE0xV2hjTgpNakV3TVRJd01UQTFOVE0xV2pBY01Sb3dHQVlEVlFRREV4RmpkWE4wYjIwdGJXVjBjbWxqY3kxallUQ0NBU0l3CkRRWUpLb1pJaHZjTkFRRUJCUUFEZ2dFUEFEQ0NBUW9DZ2dFQkFKazQ1eW9DRXRMLzZZVWM1bVovZ0o2WWUvVUUKMDZWT1U5Rk5iWk11eDM3VTNlUzdDR3N2aExCSVU1eFduSElMR1VYUTNyTk9kQTZHYXRqamsydkowTTU4TFh2KwpKZzF1MTYzSjhCSi9hSjUxRmhPTUVmSFBpWDZvSTlYUTJEWlpZUUIvV3RpTHJRazlXS2Zld1lzREIvNzBrUnNVCk0yMm1lMkJSR0cvb2gvcnY0M04wYmtodHA5QUVQd2NkWFZOcmJiWHVuRytpZEJZU0FrOUVPTXFwZXVRRDJEV1cKc3g1TmJXakNHY2YzZ1pvNjEyZCtpUysyQVNYMk1TM0pQOHppS1N1VjVQMVhTRk5qUS9peHh3S0dkVjdzWkxUMAoreFdDSXNZY0cxSXc5SXU4MDRXSVkrdWpTM20rYXliYS9ESmx0eUJ5STlSclErME8zOFQ0M2h4bGgzc0NBd0VBCkFhTkNNRUF3RGdZRFZSMFBBUUgvQkFRREFnS2tNQjBHQTFVZEpRUVdNQlFHQ0NzR0FRVUZCd01CQmdnckJnRUYKQlFjREFqQVBCZ05WSFJNQkFmOEVCVEFEQVFIL01BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQ1FsRXZ1RDVSSwozMENCR1FBenk3NGtacmtDZEdRZFBGOGRBalN4VVVJem1EdFBJWVZHTFF0RFpPTlphcmJJZmVlUVRUeStIQXJGCkVlb1g0ajVGNVlNaURUQ2FGOWgvME5EbGVmSDFmWGNsOHpvaytCQkpIbVlSMklhNVRDVk5iRjcrYXl0TkowclIKLzV3QlViYlkzR1ZoZWp4cHVOSzNXaFZEMjVZdWR0TTVPZUdZcXJjcjJSbHd5REYrd0pmWXJteFBtd1BneTBJNwpGTW02ZXhGSGFTUDYydlNUOFI5M3FQSzJiTllod0lyclcvLzN4OWRSdnF6dmFoUUxtenlvUDBBL2JBK3ZZNGhHCkZJbmhmOGtFdW5iZTh0SXk5bmVDejlhYklZUVNwQm16UFpVZG5rS2xiRlVGTVFpbkVobU1JdzY0Y0RvWXBPUW4KL2NVbXhDQlB5KzZrCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K'
```

Comment:
Do you really want to check the certificate into source control? Is it possible to use certmanager to generate it?

Reply:
Yes. This is just a dev certificate. I can change so we assume certmanager is available.

Reply:
Actually I see the cert-manager is only in kfctl_istio_dex.yaml - so what would be the preferred way of provding certs for the other options? Should cert-manager not be in all? @jlewi @richardsliu

Reply:
I have added cert-manager to all kfdefs as part of this PR